CMakeLists.txt

```diff
@@ -76,6 +76,10 @@ list(APPEND NVFUSER_SRCS
   ${NVFUSER_SRCS_DIR}/executor_utils.cpp
   ${NVFUSER_SRCS_DIR}/fusion.cpp
   ${NVFUSER_SRCS_DIR}/grouped_reduction.cpp
+  ${NVFUSER_SRCS_DIR}/id_model/id_graph.cpp
```

Comment:
looks like we forgot `id_graphs.cpp`

Reply:
Actually, that file should be removed (renamed to `id_model.cpp`)